plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestingDeltaLakeUtils.java

```diff
@@ -28,7 +28,7 @@
 import java.util.stream.Stream;
 
 import static io.trino.plugin.deltalake.DeltaLakeQueryRunner.DELTA_CATALOG;
-import static io.trino.testing.TestingConnectorSession.SESSION;
+import static io.trino.plugin.deltalake.DeltaTestingConnectorSession.SESSION;
```

Comment:
This change is required for using the new session property. Otherwise, some tests throw unknown session property.